definitions/procedures/candlepin/delete_orphaned_records_from_env_content.rb

```diff
@@ -6,7 +6,7 @@ class DeleteOrphanedRecordsFromEnvContent < ForemanMaintain::Procedure
 
       confine do
         feature(:candlepin_database) &&
-          feature(:downstream) &&
+          feature(:downstream) && feature(:downstream).satellite? &&
```

Comment:
Why not `feature(:satellite) && feature(:downstream).current_minor_version == '6.2'`

Reply:
Or better `feature(:satellite) && feature(:satellite).current_minor_version == '6.2'` if we remove the :downstream.